guide/ch_advanced_use.tex

```diff
@@ -453,6 +453,38 @@ \subsubsection{NMEA Device}
 hardware and other settings, e.g. real GPS sensor or network and WiFi
 detection.\footnote{\url{https://support.microsoft.com/en-us/windows/windows-location-service-and-privacy-3a8eee0a-5b0b-dc07-eede-2a5ca1c49088}}
 
+
+\subsection{Modernized MESA3D libraries (Software OpenGL on Windows)}
+\label{sec:ExtraData:Mesa}
+
+Stellarium uses hardware-accelerated OpenGL for rendering. On very old Windows systems without dedicated hardware, 
+or in special circumstances like virtual machines, hardware accelerated graphics may not be available, 
+and the \program{Mesa3D} software implementation of OpenGL is used. This can also be forced by the command-line option \command{-\/-mesa-mode}. 
+Rendering the graphics without hardware acceleration is of course much slower than accelerated graphics, but it's surely better than nothing.
+
+Qt's default Mesa library (Mesa 11) provides support for OpenGL~3.0 only. This prevents users of Qt5-based versions from applying dithering. 
+On Qt6, also the \program{ShowMySky} skylight model (see section~\ref{sec:skylight:ShowMySky}) is then not available. 
```

Comment:
ShowMySky doesn't depend on Qt version.

Reply:
ShowMySky is now available on Qt6 builds only, right?

Reply:
No, Qt version doesn't make any difference.

Reply:
BTW, dithering also doesn't depend on Qt version.

Reply:
OK. I remember we had discussed removing SMS from Qt5 builds as those are intended for incapable systems only. Good! 

Does dithering depend on the availability of OpenGL3.3 or not? If it does, then users of Qt5/Mesa11(OpenGL3.0) cannot apply dithering. This was written in light of SMS not available, so the only OpenGL issue to mention would have been missing dithering.

Reply:
The only factor that affects availability of high graphics mode is support for OpenGL 3.3. After all the discussions I decided to not take Qt version into account, because it wouldn't simplify anything.

High graphics mode currently defines whether we have dithering and ShowMySky.